### PR TITLE
Add tcllib 1.17 (Tcl standard library)

### PR DESCRIPTION
**Pull request by avsej**

### CODE REVIEW

**Comment by pyhalov** [comment]
pkgsend: Action payload 'COPYING' was not found in any of the provided locations...

**Comment by avsej** [comment]
@pyhalov fixed

**Comment by pyhalov** [comment]
I know nothing about tcl/tk... But shouldn't files in /usr/bin be patched to call /usr/bin/(amd64/)?tclsh . Or perhaps, files in amd64 are not necessary, as they are the same as files in non-prefixed directories?

File: components/tcl/tcllib/Makefile [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+APPS = dtplite nns nnsd nnslog page pt tcldocstrip
+
+COMPONENT_POST_INSTALL_ACTION.32 = \
+        (for app in $(APPS); do \
+          $(GSED) -i "s:/usr/bin/env tclsh:$(USRBINDIR)/tclsh:" $(PROTO_DIR)$(USRBINDIR)/$$app; \
+        done)
+
+COMPONENT_POST_INSTALL_ACTION.64 = \
+        (for app in $(APPS); do \
+          $(GSED) -i "s:/usr/bin/env tclsh:$(USRBINDIR64)/tclsh:" $(PROTO_DIR)$(USRBINDIR64)/$$app; \
```
Comment (avsej) [inline components/tcl/tcllib/Makefile]: @pyhalov is there more elegant way to format `/usr/bin/amd64/tclsh` and `/usr/bin/tclsh`?

Comment (pyhalov) [inline components/tcl/tcllib/Makefile]: You can check how it's generated, if it's using *.in files or is it just a plain file

Comment (avsej) [inline components/tcl/tcllib/Makefile]: no, it does not use substitution, paths are hardcoded in the upstream

**Pull request merged**